app/components/collection-v2/DatasetStatus.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Icon from '../chrome/Icon'
+import ExternalLink from '../ExternalLink'
+import { QriRef } from '../../models/qriRef'
+import { QRI_CLOUD_URL } from '../../constants'
+import { onClickOpenInFinder } from '../platformSpecific/DatasetStatus.TARGET_PLATFORM'
+import { ApiActionThunk } from '../../store/api'
```

Comment:
imports should be grouped with a blank line to separate in this order:

```
[module imports]

[non-component local imports]

[component imports]
```

This should be re-worked as:

```suggestion
import React from 'react'

import { QriRef } from '../../models/qriRef'
import { QRI_CLOUD_URL } from '../../constants'
import { onClickOpenInFinder } from '../platformSpecific/DatasetStatus.TARGET_PLATFORM'
import { ApiActionThunk } from '../../store/api'

import Icon from '../chrome/Icon'
import ExternalLink from '../ExternalLink'
```

Reply:
Ahh awesome, thanks for clarifying that. Will update